circle.yml

```diff
@@ -4,12 +4,14 @@ machine:
   xcode:
     version: "7.3"
 dependencies:
-  pre:
-    - brew install shellcheck
+  # pre:
+    # - brew install shellcheck
   override:
     - bundle install --jobs=4 --retry=3
 test:
   pre:
     - ulimit -n 400
   override:
+    - bundle exec danger # for quick, instant feedback of PR body and title
```

Comment:
So if I were to set it with an ID, the second run would post the test results in a separate message, but keep the PR title warning in the first one?